tests/testsuite/a_basic/tests_01_system.py

```diff
@@ -988,3 +1004,90 @@ def test018_add_list_remove_cgroup_subsystem(self):
         self.assertEqual(res['swap'], 0)
 
         self.lg('{} ENDED'.format(self._testID))
+
+    def test019_dmi_bios_info(self):
+        """ g8os-048
+        *Test case for checking on the dmi BIOS information*
+
+        **Test Scenario:**
+        #. Get the dmi bios information using g8os client
+        #. Get the information using bash
+        #. Compare dmi g8os results to that of the bash results, should be the same
+        """
+        self.lg('get dmi_bios info using linux bash command (dmidecode -t bios)')
+        dmi_bois_info = self.getDmibiosInfo(self.client)
+
+        self.lg('get dmi_bios info using g8os client')
+        bios_info = self.client.info.dmi('bios')['BIOS Information']['properties']
+        
+        keys = ['Vendor','Version','Release Date','Address','Runtime Size','ROM Size','BIOS Revision']
+        bios_info_map = dict()
+        for k in keys:
+            bios_info_map[k] = bios_info[k]['value']
+            
+        self.lg('compare g8os results to bash results')
```

Comment:
better to do a check at a time instead of comparing a whole dict to another dict .. to quickly know where is the error